### PR TITLE
Implement categories api

### PR DESCRIPTION
**Pull request by uyen3893**

### CODE REVIEW

File: queries.js [hunk 1]
```diff
@@ -117,12 +117,65 @@ const deleteBook = (req, res) => {
...
         if(err) {
-            throw error;
+            throw err;
         }
         res.status(200).send(`Delete a book with ID: ${id}`);
     })
 }
 
+//Get the list of category
+const getCate = (req, res) => {
```
Comment (phkhuong) [inline queries.js]: Should use entity's full name to make it clear for reader. Do the same for all variables below.

**Review by phkhuong — CHANGES_REQUESTED** [review]

**Comment by uyen3893** [comment]
Pleare review it.

File: queries.js (hunk 1 above)
Comment (uyen3893) [reply queries.js]: ok

**Comment by uyen3893** [comment]
Please review it

**Review by phkhuong — APPROVED** [review]

**Pull request merged**